main.py

```diff
@@ -0,0 +1,102 @@
+            text = InstagramUsers(followings=followings, followers=followers, status="new", time=str(now))
+            session.add(text)
+        session.commit()
+
+    def database_followings(self):
+        """Getting all followings of the database"""
+        session = self.create_database()
+        database_followings = session.query(InstagramUsers.followings).all()
+        return [y for x in database_followings for y in x]
+
+    def database_follower(self):
+        """Getting all followers of the database"""
+        session = self.create_database()
+        database_follower = session.query(InstagramUsers.followers).all()
+        return [y for x in database_follower for y in x]
+
+    def subscribed(self, new_followers, new_followings):
+        """Check whether the current user is in the database.If not, then add(status- subscribed)"""
+        session = self.create_database()
+        now = self.time_now()
+        for following in new_followings:  # write to database subscribed followings
+            if following not in self.database_followings():
+                text = InstagramUsers(followings=following, followers='', status="subscribed", time=str(now))
+                session.add(text)
+            session.commit()
+        for follower in new_followers:  # write to database subscribed followers
+            if follower not in self.database_follower():
+                text = InstagramUsers(followings='', followers=follower, status="subscribed", time=str(now))
+                session.add(text)
+            session.commit()
+
+    def unsubscribed(self, new_followers, new_followings):
+        """Check whether the current database user is in the current list.If not, then add(status-unsubscribed)"""
+        session = self.create_database()
+        now = self.time_now()
+        for following_l in self.database_followings():  # write to database unsubscribed followings
+            if following_l not in new_followings:
+                text = InstagramUsers(followings=following_l, followers='', status="unsubscribed", time=str(now))
+                session.add(text)
+            session.commit()
+        for follower_l in self.database_follower():  # write to database unsubscribed followers
+            if follower_l not in new_followers:
+                text = InstagramUsers(followings='', followers=follower_l, status="unsubscribed", time=str(now))
+                session.add(text)
+            session.commit()
+
+
+if __name__ == '__main__':
+    instagram_info = GetInstagramInfo(user_name, password)
+    user_id = instagram_info.get_user_id('case_iphone_lviv2019')
+    followers = instagram_info.get_followers(user_id)
+    followings = instagram_info.get_followings(user_id)
+    instagram_info.save_db(followers,followings)  # run once and commented
+    # instagram_info.subscribed(followers, followings)  # to uncomment when bd is created
```

Comment:
This is not a solution: to comment/uncomment.
Please implement an appropriate logic when everything works fine without additional manual work.

Reply:
done
We check the first line of the status, if there is `new`, if not, then create a database